doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -813,29 +813,8 @@ are applied using the tactic :n:`rewrite_strat @rewstrategy`.
 .. insertprodn rewstrategy rewstrategy
 
 .. prodn::
-   rewstrategy ::= @one_term
-   | <- @one_term
-   | fail
-   | id
-   | refl
-   | progress @rewstrategy
-   | try @rewstrategy
-   | @rewstrategy ; @rewstrategy
-   | choice {+ @rewstrategy }
-   | repeat @rewstrategy
-   | any @rewstrategy
-   | subterm @rewstrategy
-   | subterms @rewstrategy
-   | innermost @rewstrategy
-   | outermost @rewstrategy
-   | bottomup @rewstrategy
-   | topdown @rewstrategy
-   | hints @ident
-   | terms {* @one_term }
-   | eval @red_expr
-   | fold @one_term
-   | ( @rewstrategy )
-   | old_hints @ident
+   rewstrategy ::= @rewstrategy ; @rewstrategy_prefix_op
+   | @rewstrategy_prefix_op
```

Comment:
The doc should define this, right?

Reply:
I haven't yet understood the semantics of `insertprod gram1 gram2`: it meant the range of entries from gram1 to gram2.

Hopefully, this is fixed. At least, none of `make doc/tools/docgram/fullGrammar doc/tools/docgram/updated_rsts doc/tools/docgram/orderedGrammar` complains. Double-check by @jfehrle would help tough.